usl_models/usl_models/atmo_ml/model.py

```diff
@@ -222,7 +333,7 @@ def __init__(
             name="wdir10_output_cnn",
         )
 
-    def call(self, inputs: AtmoInput) -> tf.Tensor:
+    def call(self, inputs: Input) -> tf.Tensor:
```

Comment:
Should be AtmoModel.Input

Reply:
if  i have defined it as Input in the class Atmo, shouldn't keep it input here?

Reply:
done